lib/parse.js

```diff
@@ -1522,7 +1522,7 @@ function parse($TEXT, options) {
                 }
                 if (is_expand) {
                     if (!is("punc", "]")) {
-                        unexpected(); // Must be last element
+                        token_error(S.token, "Rest element must be last element");
```

Comment:
On second thought, shouldn't this be just `croak("Rest element must be last element")`?